src/lib389/lib389/cli_idm/role.py

```diff
@@ -85,7 +85,7 @@ def modify(inst, basedn, log, args, warn=True):
 
 def rename(inst, basedn, log, args, warn=True):
     dn = _get_dn_arg(args.dn, msg="Enter dn to modify")
-    _generic_rename_dn(inst, basedn, log.getChild('_generic_rename_dn'), MANY, dn, args)
+    _generic_rename_dn(inst, basedn, log.getChild('_generic_rename_dn'), Roles, dn, args)
```

Comment:
I'm a bit confused. `MANY == Roles` - at least, it should be, as the approach is used everywhere in our CLI.

Reply:
> I'm a bit confused. `MANY == Roles` - at least, it should be, as the approach is used everywhere in our CLI.

Well this has been needed to fix many other bugs around this area.  It comes down to setting "self._protected = False" otherwise with "MANY" deletes are silently ignored as protected remains True for these objects

Reply:
Sorry, and "renames", not just "deletes", also check this "protected" variable and will silently be ignored when set to True.

Reply:
What I mean is that the change in `src/lib389/lib389/cli_idm/role.py` doesn't change anything, unless I miss something...

In the beginning of the file:
```
MANY = Roles
SINGULAR = Role
```

And this PR doesn't change anything around `protected`.
But the `xfail` removal makes sense to me.